package.json

```diff
@@ -5,14 +5,14 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "server": "nodemon backend/index.js",
-    "client": "webpack --watch -d source-map",
+    "start": "node backend/index.js",
+    "build": "webpack --watch",
+    "build-dev": "webpack --watch -d source-map",
     "seed": "node database/seed.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
```

Comment:
Do we need react and react-dom in here? How about axios? I think some dependencies are missing. 
Also, babel runtime can go to dev.